internal/commands/deployment.go

```diff
@@ -307,6 +310,17 @@ func parseContainerPorts(ports []string) ([]shared.ContainerPort, error) {
 }
 
 func parseEnvVars(envVars []string) ([]shared.DeploymentConfigV3Env, error) {
+	// Envs from a Config File are parsed from urfave/cli as a single-element
+	// string slice of all the values like:
+	// []string{`[KEY1=VAL1 KEY2=VAL2 KEY3="QUOTED VAL3"]`}
+	// This stanza parses those into a proper slice of one Env per slice element
```

Comment:
Is this a problem in our altsrc implementation? there's another string slice flag that would want the same treatment. Would be cool to fix in the underlying parser instead of post-processing if possible